.editorconfig

```diff
@@ -0,0 +1,22 @@
+# See http://editorconfig.org to read about the EditorConfig format.
+# - Automatically supported by VS2017+ and most common IDE or text editors.
+# - For older VS2010 to VS2015, install https://marketplace.visualstudio.com/items?itemName=EditorConfigTeam.EditorConfig
+
+# top-most EditorConfig file
+root = true
+
+# Default settings:
+# Use 4 spaces as indentation
+[*]
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[src/overlay*{cpp,h}]
+indent_size = 3
```

Comment:
Are you sure this shouldn't be four?

Reply:
Most of overlay.* etc are 3 spaces as it was in mesa vulkan overlay.

Reply:
Ah whoops I just read `*{cpp,h}`, so nvm.